packages/contracts-bedrock/src/L1/SystemConfig.sol

```diff
@@ -273,7 +269,6 @@ contract SystemConfig is OwnableUpgradeable, ReinitializableBase, ISemver {
             l1CrossDomainMessenger: l1CrossDomainMessenger(),
             l1ERC721Bridge: l1ERC721Bridge(),
             l1StandardBridge: l1StandardBridge(),
-            disputeGameFactory: disputeGameFactory(),
```

Comment:
We could still return the DGF from the getter here, but that would require a separate `Addresses` struct since the DGF is no longer an input. I recall @maurelian had a use case for this method returning everything in OPCM, but I don't feel strongly, so will defer to you both about whether to continue returning the DGF here

Reply:
Highly prefer not returning it, this struct gets used for both inputs and outputs, if we have a value on the output then we need it on the input and it gets confusing

Reply:
I'm OK with removing this but hopefully no more values from the struct.

What I think we should do is similar to what this PR did, ie. have all the contracts for a system gettable via the SystemConfig, but have it get those addresses by querying whatever other contracts it needs to.